Makefile

```diff
@@ -358,14 +358,31 @@ androidupdate: all
 	# assets are compressed manually and stored inside the APK.
 	cd $(INSTALL_DIR)/koreader && 7z a -l -mx=9 -mfb=256 -mmt=on \
 		../../$(ANDROID_LAUNCHER_DIR)/assets/module/koreader-$(VERSION).zip * \
+		-xr!*cache$ \
+		-xr!*clipboard$ \
+		-xr!*data/dict$ \
+		-xr!*data/tessdata$ \
+		-xr!*history$ \
+		-xr!*l10n/.tx$ \
```

Comment:
We can just remove that file. Switching between Weblate and Transifex isn't a simple binary thing anyway, see https://github.com/koreader/koreader/issues/6775.